front/src/applications/editor/tools/switchEdition/components.tsx

```diff
@@ -94,6 +94,15 @@ export const SwitchEditionLeftPanel = () => {
         }}
         onSubmit={async (flatSwitch) => {
           const entityToSave = flatSwitchToSwitch(switchType, flatSwitch as FlatSwitchEntity);
+          const detectedDuplicates = filter(
+            groupBy(entityToSave.properties.ports, 'track'),
+            (v, _) => v.length > 1
+          );
+
+          if (detectedDuplicates.length) {
+            throw new Error(t('Editor.infra-errors.error-type.node_endpoints_not_unique.name'));
+          }
```

Comment:
When reading the AC in the issue, I thought we needed to disable the save button if there was a warning @bloussou  : 
- if it that's the case, we should change this
- if not and we throw the error, I think that we should maybe hide it if the user fill the form without warnings (even before submitting it)

Reply:
I think yes we should disable save button if there is an error :)

Reply:
If there is a warning, you mean ? (and we don't throw the error at all)

Reply:
We should disable the button cause we know it will trigger an error.